crates/sui-types/src/object.rs

```diff
@@ -695,20 +695,6 @@ impl Object {
         let type_tag = move_struct.type_params[0].clone();
         Ok(type_tag)
     }
-
-    pub fn ensure_public_transfer_eligible(&self) -> Result<(), ExecutionError> {
```

Comment:
Checking my understanding: gone because this is now only relevant within a Programmable Transaction and its checked on its representation of objects?

Reply:
Correct